### PR TITLE
Fetching data from json file

### PR DESCRIPTION
**Pull request by SabaMallah**

### CODE REVIEW

**Comment by SabaMallah** [comment]
> @SabaMallah is this PR? Is it ready for review?
Yes , i have created the PR.

File: app.js [hunk 1]
```diff
@@ -1,27 +1,57 @@
...
+                  <i class="fas fa-shopping-cart"></i>
+                  add to bag
+              </button>
+          </div>
+          <h3>${data[index].name}</h3>
+          <h4>$${data[index].price}</h4>
+      </article>
+  `;
+  }
+  return (productsCenterDOM.innerHTML = results);
```
Comment (imambux) [inline app.js]: Hi @SabaMallah, I have a few concerns:
- What is it exactly that is returned by this function?
- Why are you returning anything at the first place if you are not using it?

File: app.js [hunk 2]
```diff
@@ -1,27 +1,57 @@
 const productsCenterDOM = document.querySelector(".products-center");
 
 // get all products
```
Comment (imambux) [inline app.js]: please remove this comment

File: app.js [hunk 3]
```diff
@@ -1,27 +1,57 @@
...
+              </button>
+          </div>
+          <h3>${data[index].name}</h3>
+          <h4>$${data[index].price}</h4>
+      </article>
+  `;
+  }
+  return (productsCenterDOM.innerHTML = results);
+};
+getProducts();
```
Comment (imambux) [inline app.js]: Why do you need to call this function separately from a variable? If it's used just once you can call the function right away. e.g. function () {...}();

File: app.js [hunk 4]
```diff
@@ -1,27 +1,57 @@
...
+
+// productsCenterDOM.innerHTML = fetch("products.json")
```
Comment (imambux) [inline app.js]: Is this comment going to be merged with main branch? As a standard, we should not have code in the main branch that is not being used. Please remove this comment.

File: index.html [hunk 5]
```diff
@@ -45,7 +45,7 @@ <h2>our products</h2>
       </div>
       <div class="products-center">
         <!-- single product -->
-        <article class="product">
+        <!-- <article class="product">
```
Comment (imambux) [inline index.html]: please remove the comments

**Review by imambux — CHANGES_REQUESTED** [review]
Please fix the concerns

**Review by imambux — APPROVED** [review]

**Pull request merged**